phpugms_greeting_2018-10/index.html

```diff
@@ -0,0 +1,204 @@
+
+    <link rel="stylesheet" href="../reveal.js/css/reveal.css">
+    <link rel="stylesheet" href="../theme/theme.css" id="theme">
+
+    <!-- Printing and PDF exports -->
+    <script>
+        var link = document.createElement('link');
+        link.rel = 'stylesheet';
+        link.type = 'text/css';
+        link.href = window.location.search.match(/print-pdf/gi) ? '../reveal.js/css/print/pdf.css' : '../reveal.js/css/print/paper.css';
+        document.getElementsByTagName('head')[0].appendChild(link);
+    </script>
+
+    <style>
+        .reveal footer {
+            background-color: #7c83b7;
+        }
+
+        .reveal a {
+            color: #50587c;
+        }
+
+        .reveal h1:after, .reveal h2:after {
+            background: #50587c;
+        }
+    </style>
+</head>
+
+<body class="light">
+<div class="reveal">
+    <div class="slides">
+
+        <section>
+            <h2>
+                <img alt="PHP Usergroup Münster" src="../assets/phpugms-logo.svg" class="noborder" style="width: 300px">
+            </h2>
+
+            <p>
+                <a href="https://www.meetup.com/phpugms">meetup.com/phpugms</a>
+                <br/>
+                <a href="https://twitter.com/phpugms">@phpugms</a>
+                <br/>
+                Tuesday, October 16, 2018
+            <p>
+        </section>
+
+
+        <section>
+            <h2>
+                <img src="../assets/slack.svg" class="noborder" width="250px"/>
+            </h2>
+
+            <p>Room <strong>#phpugms</strong><br>
+                PHPUG Slack (<a href="https://phpug.slack.com/">phpug.slack.com</a>)<br>
+                Get your Invite (<a href="https://php.ug/slackinvite">php.ug/slackinvite</a>)
+            </p>
+        </section>
+
+        <section>
+            <h2>Slide Archive on Github</h2>
+
+
+            <p>
+                Contribute Slides and links on GitHub<br>
+                <a href="https://github.com/phpugms/">github.com/phpugms</a><br>
+                <a href="https://phpugms.github.io/">phpugms.github.io</a>
+            </p>
+        </section>
+
+
+        <section>
+            <h1>Wifi</h1>
+            <p>SSID: GS Event<br>Password: gruenspar4events</p>
+        </section>
+
+        <section>
+            <h1>Restroom </h1>
+        </section>
+
+        <section>
+            <h1>Drinks</h1>
+        </section>
+
+        <section>
+            <h2>Thank you sponsors!</h2>
+        </section>
+
+
+        <section>
+            <img src="../assets/gruenspar_logo.jpg" class="noborder"/>
+            <p>Thanks for Drinks, Wifi and Hosting</p>
+            <p>Looking for a Job? Ask the team!</p>
+        </section>
+
+        <section>
+            <h1>Upcoming Events</h1>
+        </section>
+
+
+        <section class="l-2up">
+            <h2>November Meeting</h2>
+
+            <strong>Tuesday, November 20, 2018</strong><br>
```

Comment:
Location CHECK24 Münster

Reply:
Maybe we can also announce our End-Of-Year meetup already since location and date is already fix.

Reply:
all done.